label/label.go

```diff
@@ -0,0 +1,93 @@
+package label
+
+import (
+	"context"
+	"time"
+
+	"github.com/fabric8-services/fabric8-wit/errors"
+	"github.com/fabric8-services/fabric8-wit/gormsupport"
+	"github.com/fabric8-services/fabric8-wit/log"
+
+	"fmt"
+
+	"github.com/goadesign/goa"
+	"github.com/jinzhu/gorm"
+	uuid "github.com/satori/go.uuid"
+)
+
+// APIStringTypeLabels helps to avoid string literal
+const APIStringTypeLabels = "labels"
+
+// Label describes a single Label
+type Label struct {
+	gormsupport.Lifecycle
+	ID      uuid.UUID `sql:"type:uuid default uuid_generate_v4()" gorm:"primary_key"` // This is the ID PK field
+	SpaceID uuid.UUID `sql:"type:uuid"`
+	Name    string
+	Color   string
+	Version int
```

Comment:
Just a thought, we can keep it here and use it later. Or we can introduce this when we have `update` API